embulk-guess-gzip/build.gradle

```diff
@@ -0,0 +1,160 @@
+plugins {
+    id "java"
+    id "maven-publish"
+    id "signing"
+    id "org.embulk.embulk-plugins" version "0.5.5"
```

Comment:
`0.5.5` is intentional? (not `0.6.1`)?

Reply:
Same as: https://github.com/embulk/embulk-decoder-bzip2/pull/3#discussion_r1347412132